doc/api/n-api.md

```diff
@@ -4188,6 +4189,11 @@ prevent the event loop from exiting. The APIs `napi_ref_threadsafe_function` and
 
 <!-- YAML
 added: v8.16.0
+napiVersion: 4
+changes:
+  - version: v10.17.0
```

Comment:
Still looks to me like the version: v10.17.0 needs to be updated.

Reply:
I replaced the actual version with the string "REPLACEME"